index.js

```diff
@@ -265,6 +265,8 @@ module.exports = fp(underPressure, {
   fastify: '4.x',
   name: '@fastify/under-pressure'
 })
+module.exports.default = underPressure
+module.exports.underPressure = underPressure
```

Comment:
I think this is wrong. should be module.exports.fastifyUnderPressure.

Reply:
I agree